projects/extension/src/background/index.ts

```diff
@@ -33,22 +35,61 @@ declare let window: Background
 window.manager = publicManager
 
 const l = logger("Extension")
+/**
+ * The amount of minutes that the DatabaseContentAlarm
+ * will rerun
+ */
+const periodInMinutes = 5
+
 export interface RequestRpcSend {
   method: string
   params: unknown[]
 }
 
+const setLocalStorage = async (key: string, chain: Chain) => {
```

Comment:
let's use a less generic name for this function, something like `saveChainDbContent` or similar.